routes/api/contacts.js

```diff
@@ -1,84 +1,88 @@
 import express from 'express';
-import Joi from 'joi';
-import { errorRequest } from './../../assets/errorMessages.js';
 import {
   listContacts,
   getContactById,
   removeContact,
   addContact,
   updateContact,
+  updatedStatusContact,
 } from './../../models/contacts.js';
 
 export const router = express.Router();
 
-const contactSchema = Joi.object({
-  name: Joi.string().required(),
-  email: Joi.string().required(),
-  phone: Joi.string().required(),
-});
-
 router.get('/', async (req, res, next) => {
   try {
     const response = await listContacts();
     res.json({ message: 'All contacts list', response });
   } catch (error) {
+    res.status(500).json('error - db get contact list');
     next(error);
   }
 });
 
 router.get('/:contactId', async (req, res, next) => {
+  const { contactId } = req.params;
   try {
-    const { contactId } = req.params;
     const response = await getContactById(contactId);
-    console.log(response);
-    if (!response) {
-      throw errorRequest(404);
-    }
-    res.json({ message: 'Contact by id', response });
+    return res.json({ message: 'Contact by id', response });
   } catch (error) {
+    res.status(500).json('error - db get contact by id');
     next(error);
   }
 });
 
 router.post('/', async (req, res, next) => {
+  const { body } = req;
```

Comment:
please add `Joi` validation for all `post` `put` `patch` endpoints

Reply:
added in v 1.1.0 :)